model/project_ref.go

```diff
@@ -1339,6 +1339,32 @@ func FindFirstProjectRef() (*ProjectRef, error) {
 	return &projectRef, nil
 }
 
+// FindAllMergedTrackedProjectRefsWithRestrictedProjects returns all project refs in the db
+// that are currently being tracked (i.e. their project files
+// still exist and the project is not hidden) and are not restricted projects.
+// You can optionally pass in a list of restricted projects the user has permission to view to include in the results.
+func FindAllMergedTrackedProjectRefsWithRestrictedProjects(restrictedProjectRefIds []string) ([]ProjectRef, error) {
```

Comment:
Should remove this function and tests now

Reply:
Good catch!